migration_toolkit_for_containers/mtc-migrating-vms.adoc

```diff
@@ -0,0 +1,108 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="mtc-migrating-vms"]
+= Migrating virtual machine storage
+include::_attributes/common-attributes.adoc[]
+:context: mtc-migrating-vms
+
+toc::[]
+
+You can migrate virtual machine (VM) storage _offline_, when you have the VM turned off, or _online_, when the VM is running.
```

Comment:
Could we change to 
`You can migrate virtual machine (VM) storage _offline_ (when you have the VM turned off) or _online_ (when the VM is running) `